### ISSUE TITLE
Find amount of time left before blocked in block set

### ISSUE THREAD

**Issue by iamunknown2** [proginosko/LeechBlock#8]
You can set an allocated amount of "allowance time" for a block set. However, this is not accompanied with a timer stating the amount of time you have left.

Is it possible to find the amount of time left before the block will be activated?

**Comment by proginosko** [comment]
Yes, you can add the "LeechBlock Time Left" widget to a toolbar. Right-click on any toolbar, select "Customize..." and drag the widget onto the toolbar.
